src/mlir_build.cpp

```diff
@@ -53,6 +53,7 @@ class AutoMiirHandle
 
 void check_miir_error(MiirStatus status, const std::string& miir_fn_name)
 {
+    (void)miir_fn_name; // WA till C++17 [[maybe_unused]]
```

Comment:
We already are C++17. Maybe we should use the attribute?
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/b73554749188f12984b3acb776bdc065ca1d20ec/CMakeLists.txt#L109

Reply:
[R] Just remove or comment out the name of the unused parameter
```C++
void check_miir_error(MiirStatus status, const std::string& /*miir_fn_name*/)
```